pom.xml

```diff
@@ -23,6 +23,12 @@
         <module>storage</module>
     </modules>
 
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.4</version>
```

Comment:
Use placeholder `spring.framework.version`

Reply:
Properties in parent definition are prohibited